src/server/pkg/storage/fileset/index/util.go

```diff
@@ -30,3 +34,14 @@ func Generate(s string) []string {
 	sort.Strings(fileNames)
 	return fileNames
 }
+
+// IndexPointsTo returns a list of all the chunks this index references
+func IndexPointsTo(idx *Index) (ids []chunk.ChunkID) {
```

Comment:
I like this helper function, but it probably would be cleaner to just be `PointsTo`.